lib/std/os/linux.zig

```diff
@@ -747,31 +747,31 @@ pub fn getuid() uid_t {
     if (@hasField(SYS, "getuid32")) {
         return @as(uid_t, syscall0(.getuid32));
     } else {
-        return @as(uid_t, syscall0(.getuid));
+        return @as(uid_t, @truncate(u32, syscall0(.getuid)));
```

Comment:
```suggestion
        return @truncate(uid_t, syscall0(.getuid));
```

Reply:
Thanks, wasn't sure where to place the truncate.  I adjusted all changes accordingly.

Reply:
Need, to remove the redundant `@as`, forgot about that.

Reply:
Looks fine from my side now.